src/site/content/en/blog/mainline-mensware/index.md

```diff
@@ -309,7 +309,7 @@ workbox.routing.registerRoute(navigationRoute);
 
 ## Demo
 
-<figure>
+<figure data-size="full">
```

Comment:
This is causing the border to be too big for the figure:
![Uploading Screenshot 2021-12-20 at 15.22.49.png…]()

Reply:
Without stretching it it actually looks worse... The video itself is fairly small, and by increasing the size of the video player to get the video to look good we end up getting a bunch of added white space.

![image](https://user-images.githubusercontent.com/11811422/146804231-1db9e7ec-f413-402b-9ff9-45cf4f7d6576.png)